packages/x-components/tests/e2e/cucumber/history-queries.feature

```diff
@@ -15,9 +16,9 @@ Feature: History queries component
     And   next queries are displayed
     And   related tags are displayed
     Examples:
-      | hideIfEqualsQuery | debounceInMs | maxItemsToStore | maxItemsToRender | instant  | historyQueryItem | list                                                                                                           |
-      | true              | 150          | 15              | 8                | true     | 7                | puzzle, funko, puzzle 3d, funko pop, puzzle 3d harry, funko pop iron, puzzle 3d harry potter, funko pop iron man |
-      | false             | 150          | 15              | 8                | true     | 6                | puzzle, funko, puzzle 3d, funko pop, puzzle 3d harry, funko pop iron, puzzle 3d harry potter, funko pop iron man |
+      | hideIfEqualsQuery | debounceInMs | maxItemsToStore | maxItemsToRender | instant | historyQueryItem | list                                                                                                             |
```

Comment:
Remove the extra space at the end of the word list.

Reply:
What do you mean? I can't see the error here

![image](https://user-images.githubusercontent.com/3502075/130029273-6bdcf1f8-a06f-44a1-8c15-1d5da40bd2de.png)